### DISCUSSION TITLE
Automatic Start is disabled even tho variable autostart=true

### DISCUSSION THREAD

**Discussion by FadingArabChristians** [RandomNinjaAtk/docker-lidarr-extended#34] [Q&A] [answered]
The script is not starting automatically even after force updating and triple checking that the variable autostart=true

Automatic Start Disabled, manually run using this command:
bash /config/extended/scripts/start_audio.sh

Is anyone else getting the same message or is it just me?

**Comment by RandomNinjaAtk** [comment]
Might be a bug, let me try to push a quick fix...

**Comment by RandomNinjaAtk** [comment, marked as answer]
Update should be building now that should fix it... https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/1b361766882af276a4c78c7e4f1d4831a0ab22b0

**Comment by FadingArabChristians** [comment]
Just updated, it looks like it is working. Thanks!!!
